components/common/shoppingCart/ShoppingCartModal.vue

```diff
@@ -190,18 +190,18 @@ const onCompletePurchase = () => {
 </script>
 
 <style scoped lang="scss">
-@import '@/assets/styles/abstracts/variables';
 .shopping-cart-modal {
   .shopping-cart-modal-container {
     position: fixed;
     top: 0;
     right: 0;
     height: 100%;
-    padding-top: $navbar-desktop-min-height;
+    padding-top: 80px;
```

Comment:
not needed anywhere else ?

Reply:
seems like our sidebar still use it

Reply:
Then can we keep the variable ?

Reply:
cc @preschian

Reply:
Oh, I misunderstood your question. I thought you were talking about the whole CSS class. I've moved the variable to the Tailwind config